VirtualFinland.UserAPI/src/VirtualFinland.UsersAPI/Data/Configuration/CertificationConfiguration.cs

```diff
@@ -0,0 +1,16 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using VirtualFinland.UserAPI.Models.UsersDatabase;
+
+namespace VirtualFinland.UserAPI.Data.Configuration;
+
+public class CertificationConfiguration : IEntityTypeConfiguration<Certification>
+{
+    public void Configure(EntityTypeBuilder<Certification> entity)
+    {
+        entity.Property(c => c.EscoUri).HasConversion(
+            v => string.Join(',', v),
```

Comment:
Varoituksena kertoo että `'v' may be null here.`.  Samankaltainen varoitus on myös muutamassa vastaavassa vanhemmassa muunnos-helpperissä: CollectionValueComparer, WorkPreferencesConfiguration. Taitavat olla serialisointiin liittyviä, eli jos data ei täsmää niin pitäisikin tapahtua virhe, mutta ehkä sen ei pitäisi tapahtua noissa käännös-helppereissä vaan jossain eri vastuulla?